src/html/erc20-form.html

```diff
@@ -56,10 +56,13 @@ <h2>New Transfer</h2>
       </p>
     </div>
     <footer>
+      <p data-behavior="signInstruction" style="display:none">
+        Please confirm the transaction in your connected Ethereum wallet...
+      </p>
```

Comment:
Design input needed from @corwinharrell here. I don't love that showing this causes a height adjustment.